.ci/docker/ci_commit_pins/triton.txt

```diff
@@ -1 +1 @@
-757b6a61e7df814ba806f498f8bb3160f84b120c
+5fe38ffd73c2ac6ed6323b554205186696631c6f
```

Comment:
Should this be a pin or a branch?

Reply:
Branch. To get green signal on PR, we need to bump the pin as well.